.github/workflows/build.yml

```diff
@@ -161,8 +161,13 @@ jobs:
             export CCACHE_DIR=`pwd`/ccache
             mkdir $CCACHE_DIR
             cd sources/testing
-            ./cibuild.sh -c testlist/${{matrix.boards}}.dat
+            export ARTIFACTDIR=`pwd`/../../buildartifacts
```

Comment:
Remove all `pwd`/ since other place use the relative path.

Reply:
The exported variable needs an absolute path here or we risk dumping it in a random place.